doc/source/serve/doc_code/deployment_graph_combine_two_nodes_with_passing_input_parallel.py

```diff
@@ -0,0 +1,32 @@
+import ray
+from ray import serve
+from ray.experimental.dag.input_node import InputNode
```

Comment:
i think Simon merged a shorthand diff that we can simply import `from ray.serve import InputNode` now

Reply:
yeah, it is from ray.serve.deployment_graph import InputNode